lib/portage/data.py

```diff
@@ -178,6 +178,16 @@ def _get_global(k):
 				noiselevel=-1)
 			portage_group_warning()
 
+		portage_homedir_ready = \
+				os.access(os.path.expanduser("~portage"),
+						  portage.os.R_OK | portage.os.W_OK | portage.os.X_OK)
```

Comment:
```suggestion
		portage_homedir_ready = (
				os.access(os.path.expanduser("~portage"),
						  os.R_OK | os.W_OK | os.X_OK))
```

Please don't use backslash-wrapping, or `portage.os` wrapper in the new code.